sponge/build.gradle

```diff
@@ -0,0 +1,17 @@
+plugins {
+    id 'java'
+    id 'org.spongepowered.plugin' version '0.9.0'
+}
+
+group 'io.ibj.lwcxe'
+version '1.0.0-SNAPSHOT'
+
+sourceCompatibility = 1.8
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    testCompile group: 'junit', name: 'junit', version: '4.12'
+}
```

Comment:
All unnecessary, handled by parent build.gradle.

The sponge dependency should be inserted here.